pkg/controllers/repositoryserver/repository.go

```diff
@@ -52,3 +57,20 @@ func (h *RepoServerHandler) connectToKopiaRepository() error {
 		args,
 	)
 }
+
+func (h *RepoServerHandler) getRepositoryCacheSettings() (contentCacheMB, metadataCacheMB int, err error) {
```

Comment:
Reminder to add a unit test for this along with other tests for repo server controller